src/overlays/actors/ovl_En_In/z_en_in.c

```diff
@@ -361,40 +372,40 @@ s32 func_80A7975C(EnIn* this, PlayState* play) {
     return 1;
 }
 
-s32 func_80A79830(EnIn* this, PlayState* play) {
+s32 EnIn_GetInitMode(EnIn* this, PlayState* play) {
```

Comment:
we recently called a similar thing i player as "start mode", I dont care too much about making that consistent though if you dont think it fits